src/main/java/io/github/zerthick/protectionperms/ProtectionPerms.java

```diff
@@ -78,13 +78,18 @@ public void onServerInit(GameInitializationEvent event) {
                 logger.warn("Error loading default config! Error: " + e.getMessage());
             }
         }
-
         //Load invert
         try {
-            CommentedConfigurationNode linksNode = configLoader.load().getNode("invert");
-            boolean invert = linksNode.getBoolean(false);
-            PermHandler.getInstance().init(invert);
-
+            CommentedConfigurationNode modeNode = configLoader.load().getNode("mode");
+            if (modeNode.isVirtual()) {
+                logger.warn("Config corrupt! Using defaults (whitelist mode)\nDelete " + instance.getId() + ".conf to fix");
+            }
+            String mode = modeNode.getString("whitelist");
+            if (mode.equals("whitelist")) {
+                PermHandler.getInstance().init(false);
+            } else if (mode.equals("blacklist")){
+                PermHandler.getInstance().init(true);
+            }
         } catch (IOException e) {
             logger.warn("Error loading config! Error: " + e.getMessage());
```

Comment:
We should change this from logger.warn to logger.error

Reply:
Done.